function/src/main/java/com/bloxbean/cardano/client/function/TxOutputBuilder.java

```diff
@@ -23,6 +23,7 @@ default TxOutputBuilder and(TxOutputBuilder after) {
         };
     }
 
+    @Deprecated
     default TxBuilder buildInputs(TxInputBuilder after) {
```

Comment:
@edridudi Let's not deprecate the current buildInputs(TxBuilderInput), it should just call the new buildInput() with separateChangeOutput = false. So the default behavior is still same. Also, please add some javadoc.